addons/attach/CfgMagazines.hpp

```diff
@@ -3,10 +3,32 @@ class CfgMagazines {
     class CA_Magazine;
     class B_IR_Grenade: CA_Magazine {
         ACE_Attachable = 1;
+        ACE_attachable_effect = "B_IRStrobe";
```

Comment:
To be consistent with other modules, this config etry should be renamed to something like: `ACE_attach_item`